ui/backend/Cargo.toml

```diff
@@ -27,7 +27,7 @@ syntect = "5.2.0"
 tauri-plugin-http = "2.0.0-beta"
 
 [dependencies.sqlx]
-version = "=0.7.3"
+version = "0.7.3"
```

Comment:
```suggestion
version = "0.7.4"
```

Reply:
What do you think of `0.7` instead and relaxing some other ones as well?

Reply:
That would be fine

Reply:
Ok, just have 2 of them now if you're ok with them